spec/system/project_spec.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "Project", type: :system do
+  before do
+    @author = FactoryBot.create(:user)
```

Comment:
**Suggestion** Just like other MRs should we replace the instance variable usage with `let` statements?